src/data/faq.json

```diff
@@ -376,6 +379,217 @@
+                                    "See: <a href='#warn_without_tan_how_to' target='_blank'>How can I warn other users if I have tested positive?</a>"
+                                ]
+                            },
+                            {
+                                "title": "Do I still have to call the hotline?",
+                                "anchor": "warn_without_tan_hotline",
+                                "active": false,
+                                "textblock": [
+                                    "No, as of version 3.0 of the Corona-Warn-App, it is no longer necessary to call the verification hotline."
+                                ]
+                            },
+                            {
+                                "title": "Do I need to upgrade to version 3.0, or will my current version of CWA continue to run?",
+                                "anchor": "warn_without_tan_version",
+                                "active": false,
+                                "textblock": [
+                                    "If you do not update the Corona-Warn-App to version 3.0, the verification hotline will still be available for a transitional period (until ??????h), where you can have a TAN issued.",
+                                    "After this date, either the test results will only be transmitted directly digitally from the test sites to the CWA, as before, you will then have the opportunity to warn others. Or you will receive the test result by other means. Then you can - after an update to version 3.0 - manually warn others directly using the new function \"Manage Your Tests and Warn Others\".",
+                                    "For all other functions, the version 3.0 does not differ from the previous version of the CWA.",
+                                    "See also: <a href='#warn_without_tan_how_to' target='_blank'>How can I warn other users if I have tested positive?</a>"
+                                ]
+                            },
+                            {
+                                "title": "Do I have to do a PCR test after I have warned with a positive self-test?",
+                                "anchor": "warn_without_tan_pcr_after_self_test",
+                                "active": false,
+                                "textblock": [
+                                    "From an epidemiological point of view, a PCR test is of course significantly more accurate and informative than a self-test performed by non-professionals. Therefore, a PCR test is recommended for confirmation (or verification of any false positives) of the self-test.",
+                                    "Therefore, in order to comply with local regulations, we recommend that after a positive self-test, they contact their local health department or trusted doctor to discuss further action."

```

Comment:
"they contact" => "you contact"

Reply:
Fixed in https://github.com/corona-warn-app/cwa-website/pull/3322/commits/f2b4444eace8ae8143537742d5f8e4f971a82b60, thanks!